holoviews/tests/teststreams.py

```diff
@@ -889,14 +971,14 @@ def test_selection_expr_stream_scatter_points(self):
             expr_stream = SelectionExpr(element)
 
             # Check stream properties
-            self.assertEqual(len(expr_stream._source_streams), 2)
-            self.assertIsInstance(expr_stream._source_streams[0], SelectionXY)
-            self.assertIsInstance(expr_stream._source_streams[1], Lasso)
+            self.assertEqual(len(expr_stream.input_streams), 2)
```

Comment:
Is there a particular reason to switch from `source` to `input` here?  Does that help make the terminology more consistent?

Reply:
Here was my thinking. For the `SelectionExpr` stream, it happened to be the case that all of the input streams were always attached to a single `source` element.  The `Derived` stream doesn't assume this, all it knows is that it should calculate a value based on a list of streams that come from somewhere else.  So I guess it seemed to me that using "source" here was no longer accurate.

Reply:
Thanks; makes sense to me!

Reply:
Makes sense to me too.